draft/ZEP0001.md

```diff
@@ -357,24 +333,16 @@ objects with a given prefix, which can be used to reduce the size of
 the result set and usually returns much faster than listing an entire
 bucket. It also provides an opportunity to better leverage the support
 for emulating a directory listing, via providing both a "prefix" and a
-"delimiter" parameter.
-
-The new design separates the metadata and chunk data storage objects
-via a different key prefix. All metadata objects have the key prefix
-"/meta" and all chunk data objects have the key prefix "/data". This
-allows a faster listing of all metadata keys within a given store. The
-list of metadata keys is then sufficient to provide some useful
-information to a user regarding the contents and structure of a Zarr
-hierarchy.
+"delimiter" parameter. The new design separates the metadata and chunk
+data storage objects by adding a prefix to chunk data.
 
 The new design also changes the key suffix for metadata objects. For
 example, for an array with hierarchy path "/foo/bar", the key for the
-corresponding metadata object in Zarr v2 would be
-"/foo/bar/.zarray". In Zarr v3 this becomes
-"/meta/root/foo/bar.array.json". This change means that all the names
-and types (array or group) of children of the "/foo" group can be
-discovered and listed via a single storage operation to list the
-objects with prefix "/meta/root/foo" and delimiter "/".
+corresponding metadata object in Zarr v2 would be "/foo/bar/.zarray".
+In Zarr v3 this becomes "/foo/bar/zarr.json". This change means that
+all the names and types (array or group) of children of the "/foo"
```

Comment:
In fact zarr v3 is no different from zarr v2 in this regard.  Perhaps we should just delete this section.